cmd/sink/main.go

```diff
@@ -104,6 +105,10 @@ func (sink *Sink) Receive(ctx context.Context, event cloudevents.Event) {
 	}
 	sink.Logger.Printf("successfully wrote cloudevent %s to the database\n", event.ID())
 	sink.Logger.Printf("checking if resource from cloudevent %s needs to be deleted\n", event.ID())
+	if events.IsDeleteType(event) {
```

Comment:
What about:

```suggestion
	if event.Type() == eventingSources.ApiServerSourceDeleteEventType {
```

Where "eventingSources" is "knative.dev/eventing/pkg/apis/sources".

They have the strings also hardcoded and we already depend on the knative/eventing package. This will remove the need to create our own events pacakge.